rsk-statistics/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "module",
+  "name": "rsk-statistics",
```

Comment:
consider renaming this to `rsk-testnet-deployment-stats` (and the folder name too)